### PR TITLE
Button sounds

### PR DESCRIPTION
**Pull request by Gladius1**
1st Pull request

### CODE REVIEW

**Comment by Gladius1** [comment]
So it seems it worked right?

**Comment by stubb** [comment]
Gladius1, please use git squash to combine the commits http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html so we have one commit in the end ;-)

**Comment by stubb** [comment]
Just gave it a try, it works nice. Will be merged after the small issues with the commits and the if/else are fixed!

**Comment by Gladius1** [comment]
Dang. I messed up. I somehow made a merge, whatever that is. How can I revert that merge thing.

**Comment by stubb** [comment]
"git-merge - Join two or more development histories together" ;-)
You can delete the last commit, the one with the merge and do a push with the option --force afterwards.

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,4 @@ content/gfx/atlas/*.png
 content/atlas.sql
 content/actionsets.json
 content/tilesets.json
+
```
Comment (stubb) [inline .gitignore]: I would remove this change.

**Comment by stubb** [comment]
If you need help to git squash all that together, feel free to ask!

**Comment by Gladius1** [comment]
Well in theory I know how to use squash but when I type git squash -i HEAD~2 it shows commits from before I forked. All commits made by me are gone somehow.

**Comment by LinuxDonald** [comment]
@stubb can you please send me your mail adress to linuxdonald@posteo.de please?

**Comment by stubb** [comment]
Pushed directly with 3421f08d77681055671c9454599f0caea6808b93

**Pull request closed without merging**